xdsl/ir.py

```diff
@@ -439,8 +440,18 @@ def verify(self, verify_nested_ops: bool = True) -> None:
             for region in self.regions:
                 region.verify()
 
+    # TODO replace with trait
     def verify_(self) -> None:
-        pass
+        try:
```

Comment:
I disagree with these changes. First, I don't want Operation to contain anything related to something like BinaryOperation. Second, I think this really should be `pass` as it's an Operation and we do not want to limit the kinds of operations one can create.

Reply:
agree